### PR TITLE
Sabin

### PR DESCRIPTION
**Pull request by Sabin-bhusal**
sfs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# gitpractice3
+# gitpractice3  
+## test
```
Comment (thechandanbhagat) [inline README.md:2]: Why this code?

File: example.html [hunk 2]
```diff
@@ -5,13 +5,16 @@
             some changes. Second changes
             Third changes
             Welcome to my page
-            Some changes from Chandan
+            Some changes from Chandan and sabin
             some changes
```
Comment (thechandanbhagat) [inline example.html]: remove these lines

File: example.html [hunk 3]
```diff
@@ -5,13 +5,16 @@
...
 
         </div>
 
 <div>
```
Comment (thechandanbhagat) [inline example.html]: Also remove this line

**Review by thechandanbhagat — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Sabin-bhusal) [reply README.md:2]: some explanations

File: example.html (hunk 2 above)
Comment (Sabin-bhusal) [reply example.html]: removed

File: example.html (hunk 3 above)
Comment (Sabin-bhusal) [reply example.html]: removed

**Review by thechandanbhagat — APPROVED** [review]

**Pull request merged**